src/ServiceClient/Implements/SocketClient/DeribitSocketClient.cs

```diff
@@ -33,21 +35,27 @@ public DeribitSocketClient(IOptions<DeribitOptions> options)
     protected string BookChannel => $"book.{deribitOptions.InstrumentName}.{deribitOptions.BookInterval}";
     protected string TickerChannel => $"ticker.{deribitOptions.InstrumentName}.{deribitOptions.TickerInterval}";
 
+    private void Timer_Elapsed(object? state)
+    {
+        refreshTokenTimer?.Dispose();
+        AuthenticateWithRefreshTokenAsync(CancellationToken.None).Wait();
+    }
+
     private Task SendTestAsync(CancellationToken token)
     {
         return SendAsync("public/test", new object(), token);
     }
 
     private Task SendAsync<T>(string method, T data, CancellationToken cancellationToken)
     {
-
         var request = new Request<T>
         {
             JsonRpcVersion = "2.0",
             Id = ++nextId,
             Method = method,
             Parameters = data,
         };
+        var jsonOptions = BuildOptions();
```

Comment:
This should be moved outside the method.

Here we are recreating it in every call